### PR TITLE
tp-qemu.test: migrate_multi_host.virtio_blk_data_plane

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 1266315

### CODE REVIEW

**Comment by XueqiangWei** [comment]
11:55:10 INFO | 			GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1492430111	localtime=Apr 17 11:55:11	completed successfully
11:55:10 INFO | 		END GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1492430111	localtime=Apr 17 11:55:11	
11:55:10 INFO | 	END GOOD	----	----	timestamp=1492430111	localtime=Apr 17 11:55:11	
11:55:12 INFO | Client complete
11:56:14 INFO | 			GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1492430175	localtime=Apr 17 11:56:15	completed successfully
11:56:14 INFO | 		END GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.virtio_blk_data_plane.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1492430175	localtime=Apr 17 11:56:15	
11:56:14 INFO | 	END GOOD	----	----	timestamp=1492430175	localtime=Apr 17 11:56:15

File: multi_host_migration/tests/cfg/multi_host.cfg [hunk 1]
```diff
@@ -519,6 +519,7 @@
                     drive_rerror_image1 = stop
                     bg_stress_test = autotest_control
                     test_control_file = fio.control
+                    kill_bg_stress_cmd = "killall -9 fio"
```
Comment (CongLi) [inline multi_host_migration/tests/cfg/multi_host.cfg]: For this case, it does not set 'only linux',
so how about windows?

Comment (XueqiangWei) [inline multi_host_migration/tests/cfg/multi_host.cfg]: currently, just support linux

Comment (CongLi) [inline multi_host_migration/tests/cfg/multi_host.cfg]: If so, could you please add only linux filter ?

Thanks.

Comment (XueqiangWei) [inline multi_host_migration/tests/cfg/multi_host.cfg]: updated, thank you.

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**